api/types/types.go

```diff
@@ -27,6 +27,7 @@ const (
 	DefaultVanName      string = "skupper"
 	DefaultSiteName     string = "skupper-site"
 	ClusterLocalPostfix string = ".svc.cluster.local"
+	SiteConfigMapName   string = "skupper-site"
```

Comment:
I believe the following source files could refer to this new constant:
* client_debug.go:81
* site-controller/controller.go:410
* cli/delete.go:49

Reply:
Agreed, fixed in latest update.